### PR TITLE
Add benchmarks for pseudo quotient method

### PR DESCRIPTION
**Pull request by 1e9abhi1e10**
This PR adds the benchmarks for the pseudo-remainder method.
Related Issue:
#88

See [sympy/sympy#25278](https://github.com/sympy/sympy/pull/25278)

### CODE REVIEW

File: benchmarks/polys.py [hunk 1]
```diff
@@ -71,7 +70,6 @@ def make_poly(self, n):
         g = d * (2 + x ** (n + 1) + sum([y[i] ** (n + 1) for i in range(n)]))
         return f, g, d, syms
 
-
 class _QuadraticNonMonicGCD(_GCDExample):
```
Comment (oscarbenjamin) [inline benchmarks/polys.py:73]: Why are these lines deleted? Usually there should be two lines between top level statements like `def` and `class`.

Comment (1e9abhi1e10) [reply benchmarks/polys.py:73]: I thought there should be one line between subclasses, I will correct myself.

**Comment by 1e9abhi1e10** [comment]
Closing! 
See https://github.com/sympy/sympy_benchmarks/issues/88#issuecomment-1636092386

**Comment by oscarbenjamin** [comment]
> Closing! See [#88 (comment)](https://github.com/sympy/sympy_benchmarks/issues/88#issuecomment-1636092386)

I meant that we don't need to time all 4 methods. I think that we should have benchmarks for `pdiv` and `prem` but perhaps fewer of them then there are right now for prem.
